app/database/models/tasks_list.py

```diff
@@ -25,6 +41,14 @@ def __init__(self, tasks=None):
                 raise ValueError(TypeError)
 
     def add_task(self, description, created_at, is_done=False, completed_at=None):
+        """Adds a task to the list of tasks.
+        
+        Args:
+            description: A description of the task added to the list of tasks.
```

Comment:
Try if possible to keep the line length limit for docstrings and comments at 72 characters: https://www.python.org/dev/peps/pep-0008/#id17.